Package.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version:3.1
+
+import PackageDescription
+
+let package = Package(
+  name: "Scenarios",
+  targets: [
+    Target(name: "Scenarios"),
+  ],
+  dependencies: [
+    .Package(url: "https://github.com/Quick/Quick.git", majorVersion: 1),
+    .Package(url: "https://github.com/sharplet/Regex.git", versions: Version(1, 0, 0, prereleaseIdentifiers: ["beta"])..<Version(2, 0, 0)),
```

Comment:
Once https://github.com/sharplet/Regex/pull/44 is merged and tagged, we should be able to drop the beta stuff here.